inc/registration.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * Register included providers.
+ *
+ * @param Core_Sitemaps_Provider[] $providers List of registered providers.
+ *
+ * @return Core_Sitemaps_Provider[] Updated list.
+ */
+function core_sitemaps_registration( $providers ) {
+	$providers['sitemap-index'] = new Core_Sitemaps_Index();
+	$providers['sitemap-posts'] = new Core_Sitemaps_Posts();
+	$providers['sitemap-pages'] = new Core_Sitemaps_Pages();
```

Comment:
What's the benefit of having this function on its own here instead of doing this inside the `Core_Sitemaps` class?

Reply:
Well the idea was to give an example for others to reuse, but it's not that complex and on reviewing your comment I've changed it so it's bundled with the filter as the default value.